src/components/VmDialog/index.js

```diff
@@ -365,6 +371,7 @@ class VmDialog extends React.Component {
             onChange={this.onChangeOperatingSystem}
             value={os ? os.get('id') : ''}
             data={sortedOSs}
+            fieldHelp={<FieldHelp title='Operating System' content='Type of operating system installed in the virtual machine.' />}
```

Comment:
I think it should be "installed on" not "in". I would anyway simplyfy it to:
Operating system installed on the virtual machine.